sql/sql_class.h

```diff
@@ -722,6 +722,16 @@ typedef struct system_status_var
   ulong max_statement_time_set;
   ulong max_statement_time_set_failed;
 
+  /** number of contiguous page reads */
+  ulonglong scan_pages_contiguous;
+  /** number of disjointed page reads */
```

Comment:
"contiguous" is an adjective, thus "disjointed" should probably be "disjoint" too for consistency, in code, and var name

Reply:
It's a tricky one. According to https://english.stackexchange.com/questions/122069/disjoint-toolsets-vs-disjointed-toolsets  it should be `disjointed`  unless we are talking about math sets. @georgelorch , as a native speaker, also said that in his opinion `disjointed` sounds more natural in this context.